io.github.webcamoid.Webcamoid.yml

```diff
@@ -0,0 +1,29 @@
+app-id: io.github.webcamoid.Webcamoid
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: webcamoid
+rename-icon: webcamoid
+rename-desktop-file: webcamoid.desktop
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-videos
+  - --device=all
+modules:
+  - name: webcamoid
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    post-install:
+      - install -Dm644 io.github.webcamoid.Webcamoid.metainfo.xml -t $FLATPAK_DEST/share/metainfo
+    sources:
+      - type: git
+        url: https://github.com/webcamoid/webcamoid.git
+        tag: 9.0.0
+      - type: file
+        path: io.github.webcamoid.Webcamoid.metainfo.xml
```

Comment:
> > the name of the metainfo file is still incorrect.
> 
> I don't understand, how it should be named?

here, you got the name right. unlike for the actual file in the repo.

Reply:
I got it, I forgot to rename the file.